src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react';
+import { useFetch } from '../../custom-hooks/useFetch';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+import { API_URL } from '../../constants/apiUrl';
+
+export const PeoplePage: FC = () => {
+  const { people, isLoading, errorMessage } = useFetch(API_URL);
+
+  const getPerent = (parentName: string | null) => {
+    if (parentName) {
+      return people?.find(person => person.name === parentName);
+    }
+
+    return null;
+  };
+
+  const peopleWithPerents = people ? people.map((person) => {
```

Comment:
```suggestion
  const peopleWithParents = people ? people.map((person) => {
```

Reply:
do you really need this ternary? seems like it's a bit complicated here
in your custom hook, just return empty array to avoid this validation